spec/features/user/user_show_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.describe "the user show page", type: :feature do
+  context "before logging in" do
+    before  :each do 
+      @user = FactoryGirl.create(:user)
+    end
+
+    it "cannot access the page unless loged in" do
+      visit items_path
+      visit user_path(@user)
+      expect(current_path).to eq(items_path)
+      expect(page).to have_content("You are not authorized to access this page.")
+    end
+
+    it "is valid if user is authenticated" do
+      visit login_path
+      fill_in("session_email", with: "paullorijgu@gmail.com")
+      fill_in("session_password", with: "gu")
+      click_button "Sign in"
+      visit user_path(@user)
+      expect(page).to have_content("Laul Guberson")
+    end
+  end
+
+  context "after logging in"do
+    before  :each do 
+      @user = FactoryGirl.create(:user)
+      item = FactoryGirl.create(:item)
+      order = FactoryGirl.create(:order, user_id: @user.id)
+      order_item = FactoryGirl.create(:order_item, item_id: item.id, order_id: order.id)
```

Comment:
Useless assignment to variable - `order_item`. Did you mean `order`?<br>Line is too long. [88/80]